settings.gradle.kts

```diff
@@ -47,7 +47,8 @@ include(
   "clients:filesystem-hadoop3",
   "clients:filesystem-hadoop3-runtime",
   "clients:client-python",
-  "clients:cli"
+  "clients:cli",
+  "clients:gvfs-fuse"
```

Comment:
How about using the module name "filesystem-fuse"?

Reply:
This name is too similar to `filesystem-hadoop`. However, `FUSE` and `Hadoop` are not on the same level. How about using `client-fuse` instead?

Reply:
Fuse is also a file system, right? So using filesystem-fuse looks more meaningful. 

Using "client-fuse" has two problems, 1) fuse is not just a client, it has a daemon process, 2) fuse only works with file, while our client also covers table, topic, etc.

I don't have a strong opinion if it is OK to others.

Reply:
Ok ,i use the name of `filesystem-fuse`